roles/openshift_aws/tasks/dns_dyn_zone.yml

```diff
@@ -0,0 +1,17 @@
+---
+- import_role: name='lib_dyn'
```

Comment:
Not the proper syntax.  It's not a good practice to import a role from inside another role.